ansibulled/changelog/utils.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Author: Matt Clay <matt@mystile.com>
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2020
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import logging
+import os
+import re
+
+import semantic_version
+import yaml
+
+
+LOGGER = logging.getLogger('changelog')
+
+
+def makedirs(path):
```

Comment:
Note.... this is an unsafe function.  We'd want to be very careful where it gets used... maybe remove it altogether.

Reply:
Since it mainly is there to handle pre-Python-3.5 quirks, I guess I can get rid of it. I added it when I wasn't sure what the tool should target...

Reply:
It's gone now.